tests/installation/clock_and_timezone/validate_timezone_configuration.pm

```diff
@@ -7,24 +7,24 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Handles Time and Date dialog in YaST Firstboot Configuration.
-#
+# Summary: Collects all the info about current Timezone configuration
+# and validates it against the one provided by test_data.
 # Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
 
-use base 'y2_firstboot_basetest';
+use base 'y2_installbase';
 use strict;
 use warnings;
 use scheduler 'get_test_suite_data';
 use cfg_files_utils;
 
 sub run {
-    my $test_data           = get_test_suite_data()->{time_and_date};
-    my $clock_and_time_zone = $testapi::distri->get_clock_and_time_zone();
+    my $test_data = get_test_suite_data()->{time_and_date};
+
+    my $clock_and_time_zone = $testapi::distri->get_clock_and_time_zone()->get_clock_and_time_zone_page();
```

Comment:
it looks like you wanted to just do `$testapi::distri->get_clock_and_time_zone()->get_clock_and_time_zone_page();` to ensure to be in the page. But you still need a variable for the controller to run the method below `collect_current_clock_and_time_zone_info` as you are using the page which does not have that method. am I right?

Reply:
Right! Thanks for the catch.